pom.xml

```diff
@@ -11,7 +11,7 @@
 	<properties>
 		<graphdb.version>9.0.0-TR16</graphdb.version>
 
-		<java.level>1.8</java.level>
+		<java.level>11</java.level>
```

Comment:
This needs to stay 1.8 for the time being. Other than that it looks good.

Reply:
fixed!